src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSigner.cs

```diff
@@ -32,6 +69,22 @@ public CoseSigner(AsymmetricAlgorithm key, HashAlgorithmName hashAlgorithm, Cose
             _algHeaderValueToSlip = ValidateOrSlipAlgorithmHeader();
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CoseSigner"/> class.
+        /// </summary>
+        /// <param name="key">The private key used to sign the content.</param>
+        /// <param name="signaturePadding">The padding mode.</param>
```

Comment:
```suggestion
        /// <param name="signaturePadding">The padding mode to use when signing.</param>
```

Reply:
Most of these parameters can draw inspiration from https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.pkcs.cmssigner?view=dotnet-plat-ext-6.0#properties.